src/core/plugins/aoe4.js

```diff
@@ -10,35 +11,18 @@ module.exports = {
     },
     templateFunctions: {
         async getAoe4PlayerStat(playerName) {
-            if (playerName.toLowerCase().includes("fitzbro")) {
-                playerName = 'FitzBro'
+            if (!playerName) {
```

Comment:
i added this part so that it would default to you if the command is used with no argument but another way to do it would be to add `search_id=[YOUR_STEAM_ID]` as a query param to the request